deploy/deploy.go

```diff
@@ -117,13 +106,27 @@ func (d *Deployment) String() string {
 	return string(b)
 }
 
+func checkDependencies() error {
+	bins := []string{"docker", "kubectl"}
+	var errs errlist.List
+	for _, bin := range bins {
```

Comment:
for _, bin := []string{"docker", "kubectl"}

Reply:
done, will push a commit soon